samples/Common.c

```diff
@@ -169,11 +176,13 @@ PVOID mediaSenderRoutine(PVOID customData)
     CHK(!ATOMIC_LOAD_BOOL(&pSampleConfiguration->appTerminateFlag), retStatus);
 
     if (pSampleConfiguration->videoSource != NULL) {
-        THREAD_CREATE(&pSampleConfiguration->videoSenderTid, pSampleConfiguration->videoSource, (PVOID) pSampleConfiguration);
+        THREAD_CREATE_WITH_PARAMS(&pSampleConfiguration->videoSenderTid, pSampleConfiguration->videoSource,
+                                  KVS_DEFAULT_MEDIA_SENDER_THREAD_STACK_SIZE, (PVOID) pSampleConfiguration);
```

Comment:
Shouldn't we set the stack size here using the `-DKVS_STACK_SIZE` that we obtain from the user?

Reply:
`KVS_STACK_SIZE` is a default stack size value, that will apply to most threads. However we also are employing custom stack sizes for threads that can accept much lower or higher values than the default.